interview-questions.md

```diff
@@ -12,46 +12,52 @@ Your answer:
 
 Researched answer:
 
-1. What is git? What is the difference between git and Github?
+1. What is git? What is the difference between git and Github? //version control
 
-Your answer:
+Your answer: git is local data organization and storage, accessable through your computing machine's terminal. Github is the online version, allowing multiple machines to access and edit stored.
 
-Researched answer:
+Researched answer:  GitHub, Inc. is an Internet hosting service for software development and version control using Git. It provides the distributed version control of Git plus access control, bug tracking, software feature requests, task management, continuous integration, and wikis for every project. 
+
+While Git is a tool that's used to manage multiple versions of source code edits that are then transferred to files in a Git repository, GitHub serves as a location for uploading copies of a Git repository
+
+// git hub allows multiple copies and versions of an original (which is stored and accesed online) to exist with the capability to replace or ,more likely, supplement the "source" of the code/program. git is your local machine's software that enables the user to interact with the main project once cloned to your machine's local files.
```

Comment:
Thank you for updating with your summary.  I would often err on the side of removing any copied content from the internet when submitting.  In the future, let's focus on only providing your own wording for these answers.